pgmpy/metrics/metrics.py

```diff
@@ -465,3 +465,126 @@ def SHD(true_model, est_model):
     shd = shd + (np.sum((d + d.T) > 0) / 2)
 
     return int(shd)
+
+
+def latent_admg(dag: DiscreteBayesianNetwork, observed: list) -> nx.MultiDiGraph:
+    """
+    Compute the latent‐projection ADMG L(G, S) of a DAG G onto a subset S ⊂ V.
+
+    Definition 5 (latent ADMG) (Faller et al., AISTATS 2024):
+    Let G be an ADMG with variables V and S ⊂ V . The latent ADMG L(G, S)
+    is the ADMG that contains all nodes in S,
+    all edges between nodes in S and additionally
+      1) a directed edge between X, Y ∈ S if there is a directed path from X to Y where all intermediate
+         nodes are in V \ S
+      2) a bidirected edge between X, Y if there is a (undirected) path such that every non-endpoint is a noncollider in V \ S and there are arrowheads towards
+         X and Y on the incident edges on the path.
+
+    Parameters
+    ----------
+    dag : DiscreteBayesianNetwork
+        The full DAG G on variables V (may include latent nodes).
+    observed : list of hashable
+        The subset S of variables to keep (observed nodes).
+
+    Returns
+    -------
+    nx.MultiDiGraph
+        A NetworkX MultiDiGraph over nodes in S where:
```

Comment:
Doesn't need to be a nx.MultiDiGraph. nx.DiGraph can be used to represent both directed and undirected edges. Undirected edges can be represented using two edges X -> Y and X <- Y. We already do this in `pgmpy.base.PDAG` class.